packages/create-sitecore-jss/src/init-runner.ts

```diff
@@ -5,7 +5,7 @@ import { InitializerFactory } from './InitializerFactory';
 
 export const initRunner = async (initializers: string[], args: BaseArgs) => {
   let nextStepsArr: string[] = [];
-  let appName;
+  let nextAppName;
```

Comment:
Can this file be reverted? Doesn't look like there are any changes other than stylistically